### PR TITLE
[TRWebtoon] add website support (#1623)

### LINKED ISSUE #1623: add trwebtoon
> **From:** 

**Version:** [](https://github.com/manga-download/hakuneko/commits/)

https://trwebtoon.com/
manga site in Turkish
theme: my manga reader

<details><summary>What has been done?</summary>

</details>

### PR DESCRIPTION
**Pull request by dodieboy**
Resolves #1623

### CODE REVIEW

File: src/web/mjs/connectors/TRWebtoon.mjs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import MangaReaderCMS from './templates/MangaReaderCMS.mjs';
+
+export default class RawMangaUpdate extends MangaReaderCMS {
```
Comment (27136773) [inline src/web/mjs/connectors/TRWebtoon.mjs]: The classname does not match the name of the website, i suggest to change the classname.

**Review by 27136773 — COMMENTED** [review]
Looks good, but classname should be adjusted before merge

File: src/web/mjs/connectors/TRWebtoon.mjs (hunk 1 above)
Comment (dodieboy) [reply src/web/mjs/connectors/TRWebtoon.mjs]: sry, forget to change the class name

**Review by 27136773 — APPROVED** [review]
Looks good

**Pull request merged**